src/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEnumerableFactory.cs

```diff
@@ -134,11 +153,13 @@ private static bool MatchesPattern(string expression, ReadOnlySpan<char> name, E
         internal static IEnumerable<FileInfo> FileInfos(
             string directory,
             string expression,
+            bool isNormalized,
```

Comment:
nit: add new arguments to the end of the list

Reply:
The new argument still isn't the last parameter.

Reply:
Oops sorry, I missed this one.